src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(num1, num2) {
```

Comment:
```suggestion
    add(a, b) {
```

Reply:
Don't use numbers in variables names
In this case if all that you know about this values - is that they're numbers (without any context of it's usage) and they'll be used for mathematical operations - it's better to use common math names, like `a` or `b`

Reply:
Or even better, we do know that first param - is a current result of operation, and second - is a number to add/divide/multiply/subtract, so you can use `result` and `number` here
But never use numbers in variable names